README.md

```diff
@@ -13,18 +13,22 @@ Welcome to Open Source Student Network page repository!
 1. Install `nodeJS`.Use your OS package manager to install nodeJS or follow the instructions on [nodeJS page](https://nodejs.org/en/download/).
 2. install `gatsby-cli`. Run
  `npm install --global gatsby-cli`.
-2. Clone the repository. Run `git clone http://link.git`.
-3. Install node dependencies. On repo directory run `npm install`.
+3. Clone the repository. Run `git clone https://github.com/ossn/ossn-frontend.git`.
+4. Install nvm. Follow the instructions on [nvm repo](https://github.com/creationix/nvm#installation).
+5. Run `nvm use`.
+6. Install node dependencies. On repo directory run `npm install`.
 
 ### <a name="develop"></a> Develop
-1. Open development server.
-At the repository directory run `npm run develop`
+1. Run `nvm use`.
```

Comment:
@dPliakos  What do you think about:
"At the repository directory run `nvm use`."